digital-land/submit
# Issue details/pagination

## What type of PR is this? (check all applicable)

- Feature
- Bug Fix

## Description

The primary purpose of this PR is to add the pagination to the bottom of the issue details page
It also updates the error summary heading on the issue details page to indicate the correct count of issues (as datasette limits returned rows to 1000, this was previously incorrect)

for now this has made another issue apparent which is the error summary component only shows 1k errors, this should be resolved in the future when the design is amended by Alex

## Related Tickets & Documents



- Ticket Link: https://github.com/orgs/digital-land/projects/7/views/24?filterQuery=assignee%3A%40me+-status%3ABacklog%2CUnprioritised%2C%22Ready+for+Dev%22%2C%22Scoping+for+Dev%22%2C%22Creating+Analysis%22%2C%22User+Research%2FTesting%22%2C%22In+design%22%2C%22Needs+Planning%22%2C%22In+Testing%22&pane=issue&itemId=75534984
- Related Issue #304, 
- Closes #304 

## QA Instructions, Screenshots, Recordings

I used this [page](http://localhost:5000/organisations/local-authority:LBH/tree-preservation-order/invalid%20date/3) to test the pagination. going through many pages to make sure it works as expected. 
![image](https://github.com/user-attachments/assets/c2ba701d-108e-46a3-9de0-0e363502cedc)

also looking at the error summary header to make sure it gives the correct number of entities with issues ()
![image](https://github.com/user-attachments/assets/b522a594-4997-4e14-9095-8d9c2da59c47)

## Added/updated tests?
_I moved the pagination tests into their own file as well as added some specific tests checking the rendered pagination component on the issuedeetails template_
